mocha_test/groupBy.js

```diff
@@ -330,7 +330,7 @@ describe('groupBy', function() {
         });
 
         it('handles empty object', function(done) {
-            async.groupByLimit({}, 2, function(val, next) {
+            async.groupBySeries({}, function(val, next) {
```

Comment:
This test was a dupe of the one on [L208](https://github.com/caolan/async/blob/39b20e6da268b2b737df7b068d32620b59011474/mocha_test/groupBy.js#L208). It's under `groupSeries`, so I assume I just forgot to update a copy and pasted test when I originally added the `groupBy` tests.

I was using the `groupBy` tests as a reference for some of the edge cases we usually test.